scanner/scanner.go

```diff
@@ -1,20 +1,19 @@
-package workers
+package scanner
```

Comment:
The only changed lines in this file are the ones that contain "status". Everything else is the same, just moved around. I'm sorry for the large and complex diff but it really needed some house keeping before we wrap the service up.